indigo_app/urls.py

```diff
@@ -25,6 +25,7 @@
     re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/activity$', places.PlaceActivityView.as_view(), name='place_activity'),
     re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/metrics$', places.PlaceMetricsView.as_view(), name='place_metrics'),
     re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/settings', places.PlaceSettingsView.as_view(), name='place_settings'),
+    re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/works_index', places.PlaceWorksIndexView.as_view(), name='works_index'),
```

Comment:
An underscore in a URL can usually semantically be replaced by a slash, since it's effectively scoping it. Perhaps `.../works/index.xlsx`?

Reply:
I like it!